compiler/src/iree/compiler/Codegen/Common/MergeParallelOps.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 The IREE Authors
```

Comment:
`MergeParallelOps` is misleading. Maybe call it rematerialize parallel ops? Something that suggests that ops are fused with all its consumers.